### PR TITLE
feat(playbooks): Modular detect and extract playbooks

### PR DESCRIPTION
**Pull request by topher-lo**
Replaced ad hoc playbooks with:
- List alerts: breaks up list of alerts into child workflows
- Enrich IoCs: URLs, Ipv4, file hashes for multiple threat intel providers
- Extract IoCs
- Notify user in Slack

### CODE REVIEW

File: tests/playbooks/test_playbooks.py [hunk 1]
```diff
@@ -108,7 +104,7 @@ async def test_playbook_validation(
     "filename, trigger_inputs, expected_actions",
     [
         (
-            "virustotal-quickstart.yml",
+            "virustotal_quickstart.yml",
```
Comment (topher-lo) [inline tests/playbooks/test_playbooks.py]:
```suggestion
            "tutorials/virustotal_quickstart.yml",
```

File: playbooks/detect/enrich_iocs/ipv4.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  entrypoint:
+    ref: select_ip_addresses
+    expects:
+      ipv4: list[str]
+  triggers:
+    - type: webhook
+      ref: receive_json
+      entrypoint: select_ip_addresses
+  returns:
+    ipv4: list[str]
```
Comment (daryllimyt) [inline playbooks/detect/enrich_iocs/ipv4.yml:15]: i don't think we have expectations on return values atm

File: playbooks/detect/enrich_iocs/malware_samples.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  entrypoint:
+    ref: select_malware_samples
+    expects:
+      file_hashes: list[str]
+  triggers:
+    - type: webhook
+      ref: receive_json
+      entrypoint: select_malware_samples
+  returns:
+    file_hashes: list[str]
```
Comment (daryllimyt) [inline playbooks/detect/enrich_iocs/malware_samples.yml:15]: same as above

File: playbooks/detect/enrich_iocs/urls.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  entrypoint:
+    ref: select_urls
+    expects:
+      urls: list[str]
+  triggers:
+    - type: webhook
+      ref: receive_json
+      entrypoint: select_urls
+  returns:
+    urls: list[str]
```
Comment (daryllimyt) [inline playbooks/detect/enrich_iocs/urls.yml:15]: same as above

**Review by daryllimyt — APPROVED** [review]
LGTM aside from 1 clarification on `returns`

**Pull request merged**